manifests/vhost.pp

```diff
@@ -1455,6 +1455,33 @@
 #   }
 #   ```
 #
+# lint:ignore:parameter_documentation
+# @param authz_core
+# lint:endignore
```

Comment:
The parameter does not appear in the new `REFERENCE.md`.  Is it because of this lint:ignore? Is it intended?

Reply:
done!!

Reply:
Grrr :unamused: GitHub is not happy

![screenshot of GitHub failure](https://github.com/puppetlabs/puppetlabs-apache/assets/148721/178b2d37-0f0e-4eaa-86c7-feaa9c79ef51)

I checked locally and the generated REFERENCE.md include the parameter documentation at the wrong place and include the `lint:ignore` / `lint:endignore` lines.  Removing these fix the issue and the location move where it is expected:


```suggestion
# @param authz_core
```

Reply:
I used `gssapi` as ref for documentation but seems that itself is at wrong place, because these are supposed to place under `directories` but they are not.
Have updated documentation and placed both  `gssapi` & `authz_core` under dictionary.